app/models/activity.rb

```diff
@@ -0,0 +1,3 @@
+class Activity < ActiveRecord::Base
+  belongs_to :trail
+end
```

Comment:
Just to clarify my understanding, this will create a column in our Activity model & Trail model called unique_id, with a unique id, correct?